### ISSUE TITLE
CanBroadcastBlock test fails because of Libplanet.Store.NamespaceNotFoundException

### ISSUE THREAD

**Issue by earlbread** [planetarium/libplanet#448]
```
13:50:11[@59527aE7D9b12D641732735C81A57Ef73c4e7d15][26] - Branchpoint is "5dbd63c26a614d8423a00e5accaf113bbbcca749f553b0bbb6bcff52397bcae2".
13:50:11[@230442B3CDA77E7771085936e0d4120BC1f81d30][5] - Branchpoint is "5dbd63c26a614d8423a00e5accaf113bbbcca749f553b0bbb6bcff52397bcae2".
13:50:11[@59527aE7D9b12D641732735C81A57Ef73c4e7d15][26] - Forking needed. Trying to fork...
13:50:11[@230442B3CDA77E7771085936e0d4120BC1f81d30][5] - Forking needed. Trying to fork...
13:50:11[@230442B3CDA77E7771085936e0d4120BC1f81d30][5] - Forking complete.
13:50:11[@230442B3CDA77E7771085936e0d4120BC1f81d30][5] - Trying to fill up previous blocks...
13:50:11[@59527aE7D9b12D641732735C81A57Ef73c4e7d15][26] - Append Failed. exception: Libplanet.Store.NamespaceNotFoundException: fed7fca0-c4ae-4032-860d-928c19f6cca8: The source namespace to be forked does not exist.
   at Libplanet.Store.LiteDBStore.ForkStateReferences[T](String srcNamespace, String destNamespace, Block`1 branchPoint, IImmutableSet`1 addressesToStrip) in /Users/earl/planetarium/libplanet/Libplanet/Store/LiteDBStore.cs:line 470
   at Libplanet.Blockchain.BlockChain`1.Fork(HashDigest`1 point) in /Users/earl/planetarium/libplanet/Libplanet/Blockchain/BlockChain.cs:line 812
   at Libplanet.Net.Swarm`1.SyncPreviousBlocksAsync(BlockChain`1 blockChain, Peer peer, Nullable`1 stop, IProgress`1 progress, Int64 totalBlockCount, Boolean evaluateActions, CancellationToken cancellationToken) in /Users/earl/planetarium/libplanet/Libplanet/Net/Swarm.cs:line 1381
   at Libplanet.Net.Swarm`1.AppendBlocksAsync(Peer peer, List`1 blocks, CancellationToken cancellationToken)
   at Libplanet.Net.Swarm`1.ProcessBlockHashes(BlockHashes message, CancellationToken cancellationToken) in /Users/earl/planetarium/libplanet/Libplanet/Net/Swarm.cs:line 1319
13:50:11[@59527aE7D9b12D641732735C81A57Ef73c4e7d15][26] - Something went wrong during message parsing: Libplanet.Store.NamespaceNotFoundException: fed7fca0-c4ae-4032-860d-928c19f6cca8: The source namespace to be forked does not exist.
   at Libplanet.Store.LiteDBStore.ForkStateReferences[T](String srcNamespace, String destNamespace, Block`1 branchPoint, IImmutableSet`1 addressesToStrip) in /Users/earl/planetarium/libplanet/Libplanet/Store/LiteDBStore.cs:line 470
   at Libplanet.Blockchain.BlockChain`1.Fork(HashDigest`1 point) in /Users/earl/planetarium/libplanet/Libplanet/Blockchain/BlockChain.cs:line 812
   at Libplanet.Net.Swarm`1.SyncPreviousBlocksAsync(BlockChain`1 blockChain, Peer peer, Nullable`1 stop, IProgress`1 progress, Int64 totalBlockCount, Boolean evaluateActions, CancellationToken cancellationToken) in /Users/earl/planetarium/libplanet/Libplanet/Net/Swarm.cs:line 1381
   at Libplanet.Net.Swarm`1.AppendBlocksAsync(Peer peer, List`1 blocks, CancellationToken cancellationToken)
   at Libplanet.Net.Swarm`1.ProcessBlockHashes(BlockHashes message, CancellationToken cancellationToken) in /Users/earl/planetarium/libplanet/Libplanet/Net/Swarm.cs:line 1319
   at Libplanet.Net.Swarm`1.ProcessMessageAsync(Message message, CancellationToken cancellationToken) in /Users/earl/planetarium/libplanet/Libplanet/Net/Swarm.cs:line 1273
   at Libplanet.Net.Swarm`1.<>c__DisplayClass115_0.<<ReceiveMessage>b__0>d.MoveNext() in /Users/earl/planetarium/libplanet/Libplanet/Net/Swarm.cs:line 2086```

**Comment by earlbread** [comment]
```
        04:42:52[@CC25856ad819b32073281D5b1AEEE2C1e8b284C5][16] - Append Failed. exception: System.Security.Cryptography.CryptographicException: The handle is invalid.
        
           at System.Security.Cryptography.CryptographicException.ThrowCryptographicException(Int32 hr)
           at System.Security.Cryptography.Utils.HashData(SafeHashHandle hHash, Byte[] data, Int32 cbData, Int32 ibStart, Int32 cbSize)
           at System.Security.Cryptography.MD5CryptoServiceProvider.HashCore(Byte[] rgb, Int32 ibStart, Int32 cbSize)
           at System.Security.Cryptography.HashAlgorithm.ComputeHash(Byte[] buffer)
           at Libplanet.Store.LiteDBStore.StateRefId(String namespace) in d:\a\1\s\Libplanet\Store\LiteDBStore.cs:line 577
           at Libplanet.Store.LiteDBStore.IterateStateReferences(String namespace, Address address) in d:\a\1\s\Libplanet\Store\LiteDBStore.cs:line 425
           at Libplanet.Store.StoreExtension.LookupStateReference[T](IStore store, String namespace, Address address, Block`1 lookupUntil) in d:\a\1\s\Libplanet\Store\StoreExtension.cs:line 43
           at Libplanet.Blockchain.BlockChain`1.GetStates(IEnumerable`1 addresses, Nullable`1 offset, Boolean completeStates) in d:\a\1\s\Libplanet\Blockchain\BlockChain.cs:line 264
           at Libplanet.Blockchain.BlockChain`1.EvaluateBlockAction(Block`1 block, IReadOnlyList`1 txActionEvaluations) in d:\a\1\s\Libplanet\Blockchain\BlockChain.cs:line 706
           at Libplanet.Blockchain.BlockChain`1.Swap(BlockChain`1 other, Boolean render) in d:\a\1\s\Libplanet\Blockchain\BlockChain.cs:line 1001
           at Libplanet.Net.Swarm`1.<AppendBlocksAsync>d__102.MoveNext() in d:\a\1\s\Libplanet\Net\Swarm.cs:line 1491
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
           at Libplanet.Net.Swarm`1.<ProcessBlockHashes>d__100.MoveNext() in d:\a\1\s\Libplanet\Net\Swarm.cs:line 1333
        04:42:52[@CC25856ad819b32073281D5b1AEEE2C1e8b284C5][16] - Something went wrong during message parsing: System.Security.Cryptography.CryptographicException: The handle is invalid.
        
           at System.Security.Cryptography.CryptographicException.ThrowCryptographicException(Int32 hr)
           at System.Security.Cryptography.Utils.HashData(SafeHashHandle hHash, Byte[] data, Int32 cbData, Int32 ibStart, Int32 cbSize)
           at System.Security.Cryptography.MD5CryptoServiceProvider.HashCore(Byte[] rgb, Int32 ibStart, Int32 cbSize)
           at System.Security.Cryptography.HashAlgorithm.ComputeHash(Byte[] buffer)
           at Libplanet.Store.LiteDBStore.StateRefId(String namespace) in d:\a\1\s\Libplanet\Store\LiteDBStore.cs:line 577
           at Libplanet.Store.LiteDBStore.IterateStateReferences(String namespace, Address address) in d:\a\1\s\Libplanet\Store\LiteDBStore.cs:line 425
           at Libplanet.Store.StoreExtension.LookupStateReference[T](IStore store, String namespace, Address address, Block`1 lookupUntil) in d:\a\1\s\Libplanet\Store\StoreExtension.cs:line 43
           at Libplanet.Blockchain.BlockChain`1.GetStates(IEnumerable`1 addresses, Nullable`1 offset, Boolean completeStates) in d:\a\1\s\Libplanet\Blockchain\BlockChain.cs:line 264
           at Libplanet.Blockchain.BlockChain`1.EvaluateBlockAction(Block`1 block, IReadOnlyList`1 txActionEvaluations) in d:\a\1\s\Libplanet\Blockchain\BlockChain.cs:line 706
           at Libplanet.Blockchain.BlockChain`1.Swap(BlockChain`1 other, Boolean render) in d:\a\1\s\Libplanet\Blockchain\BlockChain.cs:line 1001
           at Libplanet.Net.Swarm`1.<AppendBlocksAsync>d__102.MoveNext() in d:\a\1\s\Libplanet\Net\Swarm.cs:line 1491
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
           at Libplanet.Net.Swarm`1.<ProcessBlockHashes>d__100.MoveNext() in d:\a\1\s\Libplanet\Net\Swarm.cs:line 1339
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
           at Libplanet.Net.Swarm`1.<ProcessMessageAsync>d__99.MoveNext() in d:\a\1\s\Libplanet\Net\Swarm.cs:line 1287
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
           at Libplanet.Net.Swarm`1.<>c__DisplayClass119_0.<<ReceiveMessage>b__0>d.MoveNext() in d:\a\1\s\Libplanet\Net\Swarm.cs:line 2094
```

**Comment by earlbread** [comment]
Closed by #486.
